### PR TITLE
16# postpone a task

### PR DESCRIPTION
**Pull request by DraganKec**

### CODE REVIEW

**Review by cctraining — COMMENTED** [review]
Missing test for postponeTasks method.

**Comment by nikola-lisac** [comment]
- Move the logic to the service class
- Use Integer instead of int
- Rename method in service to getTask because it returns a single task
- Delete commented example code in the Controller and post it as a comment here on the pull request

**Comment by DraganKec** [comment]
POST
http://localhost:8080/tasks/6

Return
{
 "id": 6,
 "name": "task",
 "createdAt": "2017-08-29",
 "completed": false
}

File: src/main/java/ba/codecentric/prioritytasks/service/TaskService.java [hunk 1]
```diff
@@ -23,10 +23,25 @@ public Task saveTask(Task task) {
     }
 
     public List<Task> getAllTasks() {
+        List<Task> tttt = taskRepository.findAll();
         return taskRepository.findAll();
     }
```
Comment (BojanJakic) [inline src/main/java/ba/codecentric/prioritytasks/service/TaskService.java]: Do you need tttt ?

**Review by BojanJakic — APPROVED** [review]

**Review by nikola-lisac — APPROVED** [review]

**Review by GojkovicGoran — APPROVED** [review]

**Pull request merged**